app/src/main/java/com/example/deejai/RoomSelection/RoomAdapter.kt

```diff
@@ -37,6 +41,15 @@ class RoomAdapter(val rooms : List<Room>) : Adapter<RoomAdapter.RoomViewHolder>(
             roomName = itemView.findViewById(R.id.room_name)
             roomDistance = itemView.findViewById(R.id.room_distance)
             roomImage = itemView.findViewById(R.id.room_image) as ImageView
+            itemView.setOnClickListener(this)
+        }
+
+        override fun onClick(v: View?) {
+            val position = adapterPosition
```

Comment:
This value is not being used, please remove it.

Reply:
done